09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO <<<<<<<<<<<<<<<<<< Logger setup anew <<<<<<<<<<<<<<<<<<< (utils.py:39) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Analyzing repos/repos_400_c_round_two/rubinius___rubinius (install_repos.py:285) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running make (install_repos.py:51) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running command: make clean (install_repos.py:20) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/rubinius___rubinius_build.log (install_repos.py:21) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG /bin/sh: 1: rake: not found (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG make: *** [Makefile:21: clean] Error 127 (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running command: make distclean (install_repos.py:20) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/rubinius___rubinius_build.log (install_repos.py:21) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG make: *** No rule to make target 'distclean'.  Stop. (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running command: rm -rf autom4te.cache (install_repos.py:20) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/rubinius___rubinius_build.log (install_repos.py:21) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running command: rm -f config.status config.cache config.log (install_repos.py:20) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/rubinius___rubinius_build.log (install_repos.py:21) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running autogen (install_repos.py:57) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running command: ./autogen (install_repos.py:20) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/rubinius___rubinius_build.log (install_repos.py:21) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG /bin/sh: 1: ./autogen: not found (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running ./configure (install_repos.py:61) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running command: ./configure (install_repos.py:20) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/rubinius___rubinius_build.log (install_repos.py:21) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG find: ‘build/rubinius/codedb’: No such file or directory (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG find: ‘build/codetools/’: No such file or directory (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG find: ‘build/stdlib/’: No such file or directory (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG /usr/bin/env: ‘ruby’: No such file or directory (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running make (install_repos.py:69) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Running command: make (install_repos.py:20) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/rubinius___rubinius_build.log (install_repos.py:21) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG /bin/sh: 1: rake: not found (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius DEBUG make: *** [Makefile:18: install] Error 127 (install_repos.py:38) (rubinius___rubinius)
09/12/2024 05:38:08 PM logger_rubinius___rubinius ERROR Error: Build failed for rubinius___rubinius
Build failed for reason: make failed (install_repos.py:295) (rubinius___rubinius)
